lib/bundlex.ex

```diff
@@ -9,27 +9,29 @@ defmodule Bundlex do
   @type platform_t :: :linux | :macosx | :windows32 | :windows64 | :nerves
 
   @typedoc """
-  A map containing three fields that describe the platform.
+  A map containing four fields that describe the platform.
 
   It consists of:
   * architecture - e.g. `x86_64` or `arm64`
   * vendor - e.g. `pc`
   * os - operating system, e.g. `linux` or `darwin20.6.0`
+  * abi (optional) - application binary interface, e.g. `musl` or `gnu`
```

Comment:
Saying that `abi` key is `optional` suggests, that typespec `target` looks like `%{optional(:abi) => String.t(), ...}`, while it is `%{abi: String.t() | nil, ...}`